### ISSUE TITLE
Default to hi-res imagery only

### ISSUE THREAD

**Issue by bgirardot** [hotosm/OpenAerialMap#61]
The default should be hi-res imagery, the application is much more useful that way. I think most folks come looking for high res. defaulting to all just means that if i even realize it is an option i still have have to turn it off (each time i visit).

showing all results confounds the usefulness because of course there is low res for the area I am looking at so i have to drill down to see if what oam is saying is available is hi-res or not.

when it just displays hi-res, i can see quite quickly if my area has hi-res. this should really be the default i think.

**Comment by smit1678** [comment]
@bgirardot Agreed that this could be useful because it speeds up searching for what we assume most users are coming for - high res imagery. But this raises a challenge of how to show or alert the user that a filter has already been applied.

I'm thinking that most sites that have filter functionality usually don't pre-select filters, but I could be wrong. When pre-selecting, we would need to provide some clear indication that you are looking at a filtered view of imagery.

**Comment by bgirardot** [comment]
I think the issue you raise @smit1678 is already present, but instead of no clear indication that my proposed hires filter is selected, currently there is no indication that the "all imagery" filter is applied.

I think that points to a larger UI issue which can be solved at a later time and switching to the default hires filter would make a big improvement is usability right now would have no other impact but to improve usability immediately.

**Comment by bgirardot** [comment]
Still think this is a good idea. I do not come to OAM to find low res imagery.

**Comment by dktue** [comment]
+1

I wasn't even aware of this filter. Now openaerialmap makes sense for the first time! :-)

**Comment by bgirardot** [comment]
Since finding imagery visually is really the only way to find imagery at the moment, it is critical this UI works well.

Defaulting to "all" makes this more difficult.

How will the users know a filter is applied? The same way they discover there are filters now, except while  they discover it, it will be in a much more usable state, high res filter. As I said, 99.9% of the folks are coming here for high resolution anyway, let us default to the view best for them.

**Comment by dktue** [comment]
I also vote for this again to be done -- please default to high-res photos!
